### PR TITLE
Live Search to Career Results

### PR DESCRIPTION
**Pull request by justinxu012**

### CODE REVIEW

File: src/components/paradox/results/personality/career/filter/style.scss [hunk 1]
```diff
@@ -58,6 +58,15 @@
...
   @include theme("color", "text-color");
 }
+.searchField {
+  @extend .field;
+  @include min-width(lg) { display: none};
+}
+.searchFieldLG {
+  @extend .field;
+  display: none;
+  @include min-width(lg) { display: block};
```
Comment (tomprats) [inline src/components/paradox/results/personality/career/filter/style.scss:68]: @include min-width(lg) { display: block; }

Comment (justinxu012) [reply src/components/paradox/results/personality/career/filter/style.scss]: Ok fixed!

File: src/components/paradox/results/personality/career/filter/style.scss [hunk 2]
```diff
@@ -58,6 +58,15 @@
   @include theme("border-color", "border-color");
   @include theme("color", "text-color");
 }
+.searchField {
+  @extend .field;
+  @include min-width(lg) { display: none};
```
[the remark this answers is not in the document]
Comment (justinxu012) [reply src/components/paradox/results/personality/career/filter/style.scss]: OK fixed!

**Review by tomprats — APPROVED** [review]
Looks good, just need to bump the version

**Comment by justinxu012** [comment]
> 
Ok version bumped!

**Comment by tomprats** [comment]
@justinxu012 Was that last one from a bad merge? Tsk tsk sneaking things in again.

Also make sure you run an `npm i` after bumping the version so it bumps it in the lock file too

**Comment by justinxu012** [comment]
> 
yeah I forgot to update the testing snapshots for the changes in the PR so I had a few failing tests! Wasn't sure if I should request another review for that before merging. Good catch on the version bump, I'll push up a quick PR for that!

**Comment by tomprats** [comment]
All good, that seems reasonable. Tests are usually more ok to sneak fixes in

**Pull request merged**